charts/generic-govuk-app/templates/transition-config-cron.yaml

```
@@ -47,7 +47,9 @@ spec:
               command:
                 - sh
                 - -c
-                - "[ -d .git ] && git pull || git clone --depth=1 https://github.com/alphagov/transition-config"
+                - "[ -d transition-config ] && cd transition-config && \
+                   [ -d .git ] && git pull || \
+                   git clone --depth=1 https://github.com/alphagov/transition-config transition-config"
```

optional: could reformat this a bit for readability, maybe something like
```suggestion
                - >
                  [ -d transition-config ] &&
                    cd transition-config &&
                    [ -d .git ] &&
                    git pull ||
                  git clone --depth=1 https://github.com/alphagov/transition-config transition-config
```